pkg/controller/masteruserrecord/sync_test.go

```diff
@@ -310,7 +313,30 @@ func TestSyncMurStatusWithUserAccountStatusWhenCompleted(t *testing.T) {
 
 		// then
 		require.NoError(t, err)
-		verifySyncMurStatusWithUserAccountStatus(t, memberClient, hostClient, userAccount, mur, toBeProvisioned(), toBeProvisionedNotificationCreated())
+		expectedProvisionedTime := metav1.Now()
+		verifySyncMurStatusWithUserAccountStatus(t, memberClient, hostClient, userAccount, mur, expectedProvisionedTime, toBeProvisioned(), toBeProvisionedNotificationCreated())
+	})
+
+	t.Run("ProvisionedTime should not be updated when synced more than once", func(t *testing.T) {
+		// given
+		hostClient := test.NewFakeClient(t, mur)
+		sync, memberClient := prepareSynchronizer(t, userAccount, mur, hostClient)
+
+		// when
+		err1 := sync.synchronizeStatus() // 1st sync should not change anything, including ProvisionedTime in MUR
```

Comment:
I guess that checking the error should be here, not after all execution in the `// then` part

Reply:
Good point, I fixed the error checking